tests_new/integration_tests/aws_clients/utils.py

```diff
@@ -0,0 +1,43 @@
+import json
+import boto3
+
+from tests_new.integration_tests.aws_clients.sts import StsClient
+
+
+# it's here and not in Env test module, because it's used only here and we don't want circular dependencies
+def get_environment_access_token(client, env_uri, group_uri):
```

Comment:
I wouldn't mix that. To avoid the circular dependency I'd add access_token as a parameter in the get_group_session